zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/appliers/EventAppliers.java

```diff
@@ -327,7 +327,8 @@ private void registerJobIntentEventAppliers(final MutableProcessingState state)
     register(JobIntent.TIMED_OUT, new JobTimedOutApplier(state));
     register(JobIntent.RECURRED_AFTER_BACKOFF, new JobRecurredApplier(state));
     register(JobIntent.TIMEOUT_UPDATED, new JobTimeoutUpdatedApplier(state));
-    register(JobIntent.UPDATED, new JobUpdatedApplier(state));
+    register(JobIntent.UPDATED, 1, new JobUpdatedApplier(state));
+    register(JobIntent.UPDATED, 2, new JobUpdatedV2Applier());
```

Comment:
```suggestion
    register(JobIntent.UPDATED, 2, NOOP_APPLIER);
```

❓ Any reason you create an actual empty applier?